cvat/apps/iam/permissions.py

```diff
@@ -2018,3 +2026,44 @@ def has_object_permission(self, request, view, obj):
     def is_metadata_request(request, view):
         return request.method == 'OPTIONS' \
             or (request.method == 'POST' and view.action == 'metadata' and len(request.data) == 0)
+
+class AnalyticsReportPermission(OpenPolicyAgentPermission):
+    class Scopes(StrEnum):
+        LIST = 'list'
+        CREATE = 'create'
+
+    @classmethod
+    def create(cls, request, view, obj, iam_context):
+        permissions = []
+        if view.basename == 'analytics_reports':
+            scopes = cls.get_scopes(request, view, obj)
+            for scope in scopes:
+                self = cls.create_base_perm(request, view, scope, iam_context, obj)
+                permissions.append(self)
+
+            if job_id := request.query_params.get('job_id', None):
+                perm = JobPermission.create_scope_view(iam_context, int(job_id))
```

Comment:
The parent context may miss the object org information, so you may need to recreate the context using the job/task/project object.